tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -718,7 +717,7 @@ class FooClass
     function ($arguments) {
     echo 'a', 'b';
 },
```

Comment:
I'd say this line should've been indented as well, but that might be out-of-scope for this issue/pr.